Examples/MAX78000/Flash_CLI/main.c

```diff
@@ -184,11 +215,11 @@ void vCmdLineTask(void *pvParameters)
         if (async_read_req.rxCnt > 0) {
             /* Process character */
             do {
-                if (tmp == 0x08) {
+                if (tmp == BACK_SPACE) {
                     /* Backspace */
                     if (index > 0) {
                         index--;
-                        printf("\x08 \x08");
+                        printf("%c %c", BACK_SPACE, BACK_SPACE);
```

Comment:
I would also suggest deleting the last character in the buffer when backspace is received.  In this case we can replace the value with the null terminator to "delete" it.

Reply:
Ya sure. Null terminator would make more sense. Will do that change.